src/Monolog/RestartableInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types=1);
```

Comment:
Is that OK even if Monolog's 1.x's target is PHP 5.3 ?

Reply:
oups ;)